src/components/item/MovieItem.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import Col from 'react-bootstrap/Col';
+import styles from './styles.module.css';
+
+const MovieItem = (props) => {
+    const { id, title, genres, poster_path, release_date } = props.movie;
+    const year = new Date(release_date).getFullYear();
+    const genre = genres.join(', ');
+
+    return (
+        <Col md={4} className={styles.serialWrapper}>
```

Comment:
It's more preferable to destructure all the blocks from `styles` module as you did on l:7.